common/client-libs/gateway-client/src/client.rs

```diff
@@ -81,6 +81,9 @@ pub struct GatewayClient<C, St = EphemeralCredentialStorage> {
 
     /// Listen to shutdown messages.
     shutdown: TaskClient,
+
+    /// Number of websocket packets sent
+    sent_ws_packets: u64,
```

Comment:
if we're going to be having extra stats on the gateway client, how about wrap it in separate struct and also include sent/received packets + maybe also the data sizes?